### DISCUSSION TITLE
New line in terminal

### DISCUSSION THREAD

**Discussion by Samuon** [Eugeny/tabby#5403] [Q&A] [answered]
Hi,

I normally use the Arduino Serial Monitor to view data being sent through a serial port. In the past the serial monitor was fine because I only had to observe the data from a single COM port.

I am doing some testing where I would like to have 10 serial devices connected simultaneously. Tabby is perfect for this because I can monitor many devices in a single window. However I am having problems reading the data. 

In Arduino the data looks like this:
![image](https://user-images.githubusercontent.com/97207994/148323224-26c6c6dd-2690-4d4d-bf50-6ba54be69ed8.png)

In Tabby the data looks like this:
![image](https://user-images.githubusercontent.com/97207994/148323365-8dd838d8-73b1-47fe-8b6f-f92d9c0d4fe1.png)

I have tried all the settings in preferences but can't get the data to look like it does in Arduino.
![image](https://user-images.githubusercontent.com/97207994/148323479-a4e40ca2-edda-4f01-abf2-912c93ec4527.png)

Is there a newline setting in tabby?

Thanks,
Sam

**Comment by Samuon** [comment, marked as answer]
Update!

I can get the data to display correctly.

I didn't know that you had to reopen a new tab to apply the changes.

![image](https://user-images.githubusercontent.com/97207994/148703166-a94ec560-e822-40d6-a87e-a38b27e95886.png)

I changed the defult settings by clicking on cog>Profile and connections>Advanced>Serial and changed output newlines to Force CRLF

![image](https://user-images.githubusercontent.com/97207994/148703214-6b365e94-bf79-487c-8a70-b9c91ed98dbf.png)

**Comment by lawweiliang** [comment]
@Samuon  
Please close the discussion if the question was answered. 
It would be helpful to make the discussion section clean. 

Click `mark as answered` and `close the discussion`. 😄 

![image](https://user-images.githubusercontent.com/34936929/228098333-222ac3d4-4c39-4c26-a2a8-52906647e00e.png)
